--- madgraph5/madgraph/interface/common_run_interface.py	2014-05-16 05:28:15.000000000 -0700
+++ madgraph5/madgraph/interface/common_run_interface.py	2014-08-01 14:20:01.000000000 -0700
@@ -1464,8 +1464,8 @@
         """ find a valid run_name for the current job """
 
         name = 'run_%03d'
-        data = [int(s[4:6]) for s in os.listdir(pjoin(me_dir,'Events')) if
-                        s.startswith('run_') and len(s)>5 and s[4:6].isdigit()]
+        data = [int(s[4:]) for s in os.listdir(pjoin(me_dir,'Events')) if
+                        s.startswith('run_') and len(s)>5 and s[4:].isdigit()]
         return name % (max(data+[0])+1)
